docs/userGuide/syntax/lists.md

```diff
@@ -109,3 +109,98 @@ first number
 - [x] Item 4
 - ( ) Item 5
 </div>
+
+****Customizing the list appearance:****
+
+**To customize list icons, add the configuration `{icon="icon-name"}` either after a specific list item or at the end of the entire list.**
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">markdown</variable>
+<variable name="code">
+* item 1 {icon="fas-fa-times"}
+* item 2
+  * item 2.1 {icon="fas-fa-check"}
+  * item 2.2
+* item 4 {icon="fas-fa-trophy"}
+* item 5 {icon="glyphicon-education"}
+* item 6 {icon="mif-perm-media"}
+* item 7 {icon="zzz"}
+* item 8 {icon="glyphicon-education"}
+* item 9 {icon="octicon-git-pull-request"}
+  </variable>
+</include>
+
+**The icon's appearance can be further customized by adding a `class` attribute.**
+
+<div id="main-example-markbind">
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">markdown</variable>
+<variable name="code">
+* item 1 {icon="fas-fa-times" class="text-danger"}
+* item 2 
+  * item 2.1 {icon="fas-fa-check" class="text-primary"}
+  * item 2.2
+* item 3 {icon="fas-fa-home" class="text-warning"}
+* item 4 {icon="fas-fa-trophy" class="text-warning"}
+* item 5 {icon="glyphicon-education" class="text-warning"}
+* item 6 {icon="mif-perm-media" class="text-warning"}
+* item 7 {icon="zzz"}
+* item 8 {icon="glyphicon-education" class="text-warning"}
+* item 9 {icon="octicon-git-pull-request" class="text-warning"}
+</variable>
+</include>
+
+</div>
```

Comment:
Give an example of a lengthier class attribute e.g., `class="badge rounded-pill bg-danger text-white"`. Badges would look nice as list bullet points, but not sure if it will work in the current implementation.

On a related note, perhaps we can consider a `round`/`square` attributes to let users style the icon as a circle or a square (with a bg color)? No need to do in this version though.